### PR TITLE
[Impeller] Add a note about Graphite to the FAQ.

### PR DESCRIPTION
**Pull request by chinmaygarde**
Action item from go/impeller-faq-followups.

### CODE REVIEW

**Review by jonahwilliams — APPROVED** [review]
LGTM

File: impeller/docs/faq.md [hunk 1]
```diff
@@ -275,3 +275,21 @@ the name seemed apt. It's also an internal component and perhaps 30 seconds were
...
+### How does Graphite (a new GPU backend for Skia) relate to Impeller?
+
+Graphite is a new backend the Skia team is building to replace its legacy
+renderer (called Ganesh).
+
+Similar to Impeller, Graphite is designed to be optimized for modern GPU APIs
+(e.g Metal, Vulkan, Dawn). It aims to reduce the CPU cost of recording commands
+while taking advantage of newer GPU features.
+
+Improvements are being made in Graphite to allow for easier pre-compilation of
```
Comment (sethladd) [inline impeller/docs/faq.md:288]: this paragraph would get out of date, right?

Perhaps a phrasing here is "One of the goals of Graphite is to allow for easier pre-compilation of shaders at startup time." ? WDYT?

Comment (chinmaygarde) [reply impeller/docs/faq.md]: Done. PTAL.

**Review by zanderso — APPROVED** [review]

File: impeller/docs/faq.md [hunk 2]
```diff
@@ -275,3 +275,21 @@ the name seemed apt. It's also an internal component and perhaps 30 seconds were
...
+Similar to Impeller, Graphite is designed to be optimized for modern GPU APIs
+(e.g Metal, Vulkan, Dawn). It aims to reduce the CPU cost of recording commands
+while taking advantage of newer GPU features.
+
+One of the goals of Graphite is to allow for easier pre-compilation of shaders
+at startup time. But it still aims to support Skia’s general 2D API and has the
+same spec. requirements. The design decisions made to support those requirements
+make offline shader compilation impossible.
+
+Right now, Flutter has no plans to use Graphite. However, we, the Flutter team,
```
Comment (sethladd) [inline impeller/docs/faq.md:293]: "As of August 2024, Flutter has ..."

**Comment by sethladd** [comment]
LGTM with small tweak. Thank you!

**Review by sethladd — APPROVED** [review]

**Pull request merged**